cpp/src/io/parquet/parquet_common.hpp

```diff
@@ -34,7 +34,6 @@ enum Type : int8_t {
   BOOLEAN              = 0,
   INT32                = 1,
   INT64                = 2,
-  INT96                = 3,  // Deprecated
```

Comment:
same comment here...this should stay

Reply:
Thanks for the valuable suggestions, Ed. Reverted!